marine.c

```diff
@@ -915,4 +830,8 @@ static void reset_epan_mem(capture_file *cf, epan_dissect_t *edt, gboolean tree,
     cf->epan = marine_epan_new(cf);
     epan_dissect_init(edt, cf->epan, tree, visual);
     cf->count = 0;
+    // Sadly, wireshark caches ether name resolving even when resolving is disabled
+    // (Practically caching hex representation of macs)
+    // So we clear its cache ourselves
+    clear_wmem_map(get_eth_hashtable());
```

Comment:
This is very cool!
Can you create a list of more tables like this (and see if they can be cleared too)? Surely there are heavier ones for IPs, ports, conversations, etc :)

Also, it would be interesting to find out how heavy this call really is.

Reply:
I can and I have.
All the other address resolving cache tables are propery disabled by `disable_name_resolution()` and stay on 0 all the time.
I'll add some benchmark results to present the costs.

Reply:
That's awesome. Where does the +15 bytes per packet come from then? Maybe we should try taking another look at it and have this PR completely close the issue.